### ISSUE TITLE
annotation missing?

### ISSUE THREAD

**Issue by ValWood** [pombase/website#682]
I can't see why this isn't on the page?
It isn't in the logs and it's more specific than the 2 NAS (child of both)

/GO="aspect=P; term=iron import into the mitochondrion; GOid=GO:0048250; evidence=ISS; db_xref=GO_REF:0000024; with=SGD:S000003669; date=20160228"

SPAC4G8.08

**Comment by ValWood** [comment]
also 
/GO="aspect=P; term=iron import into the mitochondrion; GOid=GO:0048250; evidence=IC; db_xref=GO_REF:0000111; from=GO:0005381|GO:0005739; date=20171122"

this was present last week. The number of genes annotated to "mitochindrial transport" has dropped by 3. this is no longer annotated to mitochondrial transport.

**Comment by ValWood** [comment]
these are the other 2

/GO="aspect=P; term=mitochondrial citrate transmembrane transport; GOid=GO:0006843; evidence=ISS; db_xref=GO_REF:0000024; with=SGD:S000000495; date=20040629"
/GO="aspect=P; term=iron import into the mitochondrion; GOid=GO:0048250; evidence=ISS; db_xref=GO_REF:0000024; with=UniProtKB:Q03218; date=20040507"

**Comment by mah11** [comment]
Short answer: wait a day or two and it will sort itself.

Gory details: I thought I saw log messages about GO:0048250 yesterday, but now I don't see a set of logs from the 15th or 16th. (If there was a build that didn't get saved, then I wasn't dreaming and all is well, as what I saw would mean the log checks were working.) But anyway, I don't think this is due to a bug on our end. Last week, the GO:0048250 annotations used GO:1990925, but the terms have since been merged. On Wednesday, I ran checks before committing a GAF update to GO, and the owltools checks flagged GO:1990925 because of the merge. That, in turn, got confusing because the checks used the latest editors' version of GO, and the merge was only in that version of GO; it hadn't propagated to other GO files, presumably including the one Kim loads into Chado.

I updated GO:1990925 to GO:0048250 in Artemis on Wednesday anyway, on the assumption that everything would wend its way through the various update pipelines and self-resolve. I still think it probably will sort itself out once our load troubles go away. The GO merge is now "live" in various released GO files, so the annotations should load and reach the web in due course.
